Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
Are you sure you need to do that?
It works for me:
https://github.com/pierrre/imageserver/pull/5
https://travis-ci.org/pierrre/imageserver/builds/49433874
https://travis-ci.org/pierrre/imageserver/jobs/49433875